zboss/zboss_release_notes/release_notes.rst

```diff
@@ -0,0 +1,58 @@
+.. _zboss_changelog:
+
+Changelog
+#########
+
+All notable changes to this project are documented in this file.
```

Comment:
Missing notes:
 - Removal of the `zb_void_t` type.
 - Fixing the factory new status issue.
 - Callback passed as parameter in APSME API.
 - Is `ZB_JOINING_LIST_SUPPORT` and `CONFIG_ZB_ENHANCED_BEACON_SUPPORT` feature mentioned somewhere or is it a bug in the release script?

Reply:
Just to recap: we agreed to add three release notes. The `ZB_JOINING_LIST_SUPPORT` and `CONFIG_ZB_ENHANCED_BEACON_SUPPORT` are new macros, that allows to enable/disable functionalities, that were already included in previous nrfxlib's ZBOSS.